### ISSUE TITLE
Improve code compatibility

### ISSUE THREAD

**Issue by Sunnyyoung** [Sunnyyoung/WeChatTweak-CLI#2]
- [x] Migrate from Makefile installation compatibility
- [x] Upgrade compatibility

**Comment by n00d1e5** [comment]
```
Error: Invalid cask: /usr/local/Homebrew/Library/Taps/sunnyyoung/homebrew-repo/Casks/swift-runtime-support.rb
These libraries are included in the OS starting with macOS Mojave 10.14.4.
Error: Cannot tap sunnyyoung/repo: invalid syntax in tap!

**Comment by Sunnyyoung** [comment]
@n00d1e5 Fixed in https://github.com/Sunnyyoung/homebrew-repo/commit/2635ee320b68753be43f796390d99e7d866d82c6
